osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -102,6 +105,9 @@ private void load()
                 CacheDrawnFrameBuffer = true;
                 RelativeSizeAxes = Axes.Both;
 
+                titleBinding = localisation.GetUnicodePreference(metadata.TitleUnicode, metadata.Title);

```

Comment:
you should probably be binding to both of these separately. what happens if the title changes but the artist doesn't?

Reply:
then this needs to be fixed in various places

Reply:
quite possibly, yeah. I think we should probably have the artist/title stuff handled inside BeatmapMetadata (ie. give it a ctor localisationManager argument) then have BeatmapMetadata expose a `Bindable<string> DisplayTitle` which it culminates the artist/title/events into